docs/docs/external-volumes

```diff
@@ -62,15 +62,21 @@ You specify an external volume in the app definition of your Marathon app. [Lear
 }
 ```
 
-In the `volumes` section of app definition above:
+In the app definition above:
+
+- `containerPath` specifies where the volume is mounted inside the container. This path is configurable, but defaults to `data/`. See [the REX-Ray documentation on data directories](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/#data-directories) for more information.
+
+- The `external.driver["dvdi/driver"]` option specifies which Docker volume driver to use for storage. If you are running Marathon on DCOS, this value should always be `rexray`. [Learn more about REX-Ray](https://rexray.readthedocs.org/en/v0.3.2/user-guide/schedulers/).
+
+- You can specify additional options with `container.volumes[x].external.options[optionName]`.
```

Comment:
@jdef One final question: how can the user learn what other options are available?